lib/streamlit/config.py

```diff
@@ -589,6 +589,21 @@ def _server_port() -> int:
     return 8501
 
 
+_create_option(
+    "server.scriptHealthCheckEnabled",
+    visibility="hidden",
+    description="""
+    The endpoint used for checking if a script loads succefully. On success,
+    the endpoint will return a 200 HTTP status code. On failure, the endpoint
+    will return a 503 HTTP status code.
```

Comment:
This description is now slightly stale since it seems like the endpoint name is configurable, but we changed to having a fixed endpoint that can be enabled/disabled.